x/starname/types/naming.go

```diff
@@ -3,13 +3,9 @@ package types
 // Module names
 const (
 	// ModuleName is the name of the module
-	ModuleName = "domain"
+	ModuleName = "starname"
 	// DomainStore key defines the store key used to store domains information
-	DomainStoreKey = "domain"
-	// AccountStoreKey defines the store key used to store account information
-	AccountStoreKey = "account"
-	// IndexStoreKey defines the store key used to store indexing information
-	IndexStoreKey = ModuleName + "index"
+	DomainStoreKey = "starname"
 	// RouterKey defines the path used to interact with the domain module
 	RouterKey    = ModuleName
 	QuerierAlias = "starname"
```

Comment:
Would making `QuerierAlias = "domain"` be expeditious?

Reply:
I'm not understanding where htis is coming from... I'll check now...

Reply:
> `curl -X POST "http://localhost:1317/starname/query/resolve" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"starname\":\"dave*iov\"}"`
> 
> yields
> 
> ` "error": "unknown request: no custom querier found for route starname"`
> 
> @fdymylja?

"custom" subpath is missing, full path is: custom/module-name/query-path